src/main/java/teammates/common/util/Utils.java

```diff
@@ -33,5 +35,27 @@ public static Gson getTeammatesGson() {
                 .setPrettyPrinting()
                 .create();
     }
-
+    
+    /**
+     * This method returns a string representing a list of participants separated by delimiters.
```

Comment:
Rather than saying what it returns, it might be better to say what it does. For example, it converts a list of FeedbackParticipantType objects into a string, separated by the delimeter.